recipe/meta.yaml

```diff
@@ -10,7 +10,9 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 0
+  number: 1
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
```

Comment:
@peastman Are you sure we want a CPU version here?

Reply:
I'm just followinh trendsfrom pytorchand

Reply:
tensorflow

Reply:
Any reason not to have a CPU version?  The plugin doesn't require a GPU.

Reply:
Oh i see. Then the reason is that we haven't really tested how the ABI between CPU / GPU works out.

It seemed to me like it doesn:
https://github.com/openmm/openmm-torch/blob/master/CMakeLists.txt#L135

Maybe i'm reading something wrong?

I guess the ABI sin't tested for compatibility between CPU and GPU.

Reply:
it likely is, we should then fix how we export pinnings.

Reply:
If you tell OpenMM to use the CUDA platform, it assumes PyTorch has CUDA support and explicitly moves all calculations to the GPU.  If you use any other platform (OpenCL, CPU, or Reference), it doesn't assume anything about PyTorch.

Reply:
I guess *my* point is that I noticed that this recipe was pinned against an old version of PyTorch. my main goal is to "migrate" this forward.

Typically, we add the cpu/gpu architecture in the bulid number. I decided to add this to recipe.

Other than that, I don't know much about openmm, so that is ultimately up to do.

Reply:
For an explanation of how the cpu/cuda build strings can be used see: 

https://github.com/conda-forge/pytorch-cpu-feedstock/issues/71#issuecomment-954054683

Reply:
@jaimergp does it look ok to you?

Reply:
My point was more about how OpenMM only ships a GPU variant, so the full `cudatoolkit` stack will be downloaded anyway. So being able to choose the CPU variant looked moot here. You can also choose the backends in PyTorch, right?

Reply:
(And by the way, thank you so much for this, Mark!)

Reply:
Are there cases where not having the CPU variant could cause problems?  For example, if a user has also installed a CPU-only version of some other package, and that package specifies that it requires the CPU-only version of PyTorch?

Reply:
Hm, that'd depend on the `run_constrained` metadata but yes, there's a chance for conflict, you are right.

Reply:
Ok, it seems safest to include it.  Shall I merge this?

Reply:
Yep, I'll do it!